src/hyperdrive-cli/commands/stakewise/status/get-node-status.go

```diff
@@ -2,11 +2,28 @@ package status
 
 import (
 	"fmt"
+	"strings"
+	"unicode"
 
 	"github.com/nodeset-org/hyperdrive/hyperdrive-cli/client"
 	"github.com/urfave/cli/v2"
 )
 
+func camelToSnake(input string) string {
```

Comment:
I'd suggest having more formatted human-readable strings for each status, like
`active_ongoing` => `Active`
`exited_slashed` => `Slashed`
`withdrawal_pending` => `Exited Safely (Withdrawal Pending)`
instead of just CamelCasing the raw status types. We can workshop what they should be if you'd like.